### PR TITLE
RS setup scripts updates

### PR DESCRIPTION
**Pull request by basiliskus**
# RS setup scripts updates

- Consolidated RS setup scripts into one to simplify process
- Updated RS Setup instructions
- Improved handling of shared variables by adding .env file template
- Consolidated readme files into one to have one place where scripts are documented
- Refactored scripts in `/scripts` for separation of concerns, code reusability, maintenance and discoverability
- Updated to reference files with absolute paths so scripts can be moved and run anywhere

## Issue

#1488

### CODE REVIEW

File: scripts/setup/setup-reportstream.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+# Check requirements for this script
+check_env_vars RS_HOME
+check_installed_commands yq
+
+# Need to CD to the RS codebase local working copy to run the gradlew commands
+CURRENT_DIR=$(pwd)
+cd "$RS_HOME" || exit
+
+# source "./prime-router/.vault/env/.env.local"
```
Comment (pluckyswan) [inline scripts/setup/setup-reportstream.sh:14]: Not sure what this comment means

File: scripts/README.md [hunk 2]
```diff
@@ -1,3 +1,236 @@
...
+
+## Available Scripts
+
+### submit.sh
+
+Sends a HL7 message to RS and tracks its status throughout the flow until final delivery. When running locally, it grabs the snapshots of the file in azurite after converting to FHIR, after applying transformations in TI, and after converting back to HL7; and it copies those files to the same folder where the submitted file is. If running in a deployed environment we currently don't have a way to download the files from Azure, but the script will print the relative path for the files in the blob storage container.
+
+#### Requirements
+
+- hurl
```
Comment (pluckyswan) [inline scripts/README.md:30]: Might be helpful to add links to installation pages for the requirements

File: scripts/setup/setup-reportstream.sh (hunk 1 above)
Comment (basiliskus) [reply scripts/setup/setup-reportstream.sh:14]: Missed that. We don't need it so will remove it

File: scripts/README.md (hunk 2 above)
Comment (basiliskus) [reply scripts/README.md:30]: I added the links to the Resources section at the bottom so we don't have multiple repeated URLs

**Review by saquino0827 — APPROVED** [review]

**Pull request merged**